### ISSUE TITLE
xml - ignoreWhite problem

### ISSUE THREAD

**Issue by delta04** [ruffle-rs/ruffle#2332]
XML.prototype.ignoreWhite = true

Flash player can read a xml-file written with white spaces 
<Book>
<thumbs>
<thumb>pages/page1.swf</thumb>
<thumb>pages/page2.swf</thumb>
</thumbs>
</Book>

but ruffle only reads it correct, when the file is written 

<Book><thumbs><thumb>Pages/page1.swf</thumb><thumb>Pages/page2.swf</thumb</thumbs></Book>

**Comment by delta04** [comment]
Ups - let me try again
Flash player can read an xml-file written with white spaces 
`<`Book`>`
`<`thumbs`>`
`<`thumb`>`pages/page1.swf`<`/thumb`>`
`<`thumb`>`Pages/page2.swf`<`/thumb`>`
`<`/thumbs`>`
`<`/Book`>`
but ruffle only reads it correct, when the file is written 

<`Book`>``<`thumbs`>``<`thumb`>`pages/page1.swf`<`/thumb`>``<`thumb`>`pages/page2.swf`<`/thumb`<`/thumbs`>``<`/Book`>

**Comment by Herschel** [comment]
Thanks for the report! Could you attach or pastebin the XML that isn't read correctly for testing so that GitHub doesn't break the formatting? Thank you!

**Comment by delta04** [comment]
Hi
I have been testing some of the files from my site to be able to give a
more exact description of the problem. However, it appears that it is not
an ignore.white issue.
Ruffle does that part as it should, although it is a bit more "sensitive"
than flash.

The problems I found with ruffles reading the xml-files and executing
functions seems to be caused by something else:
          * if(success)  -> do some action   ->else -> *

I have attached the files below to explain and to show the problems.

Thanks


Den fre. 8. jan. 2021 kl. 20.51 skrev Mike Welsh <notifications@github.com>:

**Comment by Herschel** [comment]
Hi, unfortunately your files were not attached.
Could you please attach them as a ZIP directly to the GitHub issue: https://github.com/ruffle-rs/ruffle/issues/2332
You can drag the ZIP onto the comment box.
Thank you!

**Comment by delta04** [comment]
[ignorewhite_2332.zip](https://github.com/ruffle-rs/ruffle/files/5823132/ignorewhite_2332.zip)

**Comment by delta04** [comment]
I think this issue should be raised as an xml.onload-issue instead, so I close it here
